src/router/handlers/open.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @swagger
+ * /org/:
+ *  get:
+ *    description: Use to request organization
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        schema:
+ *          type: object
+ *          properties:
+ *            organizationName:
+ *              type: string
+ *              example: Mount Allison
+ */
+let getOrganization = async (req, res) => {
```

Comment:
This makes sense. Initially, I was confused why the `create` and `update` were in a separate file from `get`, but separating based on user permissions seems sensible too.

Reply:
Haha yeah, it is a bit different than our previous approach.

I've found that when developing endpoints I am usually more concerned with their security privilege, than whether they're grouped with other endpoints that have similar paths.